USAGE.md

```
@@ -0,0 +1,200 @@
+| `registerPath` | `string` | ✓ | main用コード出力ディレクトリ |
+| `rendererPath` | `string` | ✓ | renderer用コード出力ディレクトリ |
+| `ignores` | `string[]` | | 除外するファイルパターン |
+| `eventDirPath` | `string` | | イベント定義ディレクトリ |
+| `unwrapResults` | `boolean` | | Result型をアンラップするか（デフォルト: false） |
+| `customErrorHandler` | `object` | | カスタムエラーハンドラー設定 |
+| `validatorConfig` | `object` | | バリデーター設定 |
+| `errorHandlerConfig` | `object` | | 個別エラーハンドラー設定 |
+
+## 出力ファイル構造
+
+```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)